products/bio-formats/downloads/index.html

```diff
@@ -99,67 +100,19 @@ <h3 class="text-center">Tools</h3>
-                            </tr>
-                            <tr>
-                                <td>Bio-Formats API library</td>
-                                <td><a href="https://downloads.openmicroscopy.org/bio-formats/5.5.0/artifacts/formats-api.jar">formats-api.jar</a></td>
-                            </tr>
-                            <tr>
-                                <td>Bio-Formats BSD library</td>
-                                <td><a href="https://downloads.openmicroscopy.org/bio-formats/5.5.0/artifacts/formats-bsd.jar">formats-bsd.jar</a></td>
-                            </tr>
-                            <tr>
-                                <td>Bio-Formats GPL library</td>
-                                <td><a href="https://downloads.openmicroscopy.org/bio-formats/5.5.0/artifacts/formats-gpl.jar">formats-gpl.jar</a></td>
-                            </tr>
-                            <tr>
-                                <td>Bio-Formats Plugins for ImageJ</td>
-                                <td><a href="https://downloads.openmicroscopy.org/bio-formats/5.5.0/artifacts/bio-formats_plugins.jar">bio-formats_plugins.jar</a></td>
-                            </tr>
-                            <tr>
-                                <td>Bio-Formats testing framework</td>
-                                <td><a href="https://downloads.openmicroscopy.org/bio-formats/5.5.0/artifacts/bio-formats-testing-framework.jar">bio-formats-testing-framework.jar</a></td>
-                            </tr>
-                            <tr>
-                                <th colspan="2">Fork</th>
-                            </tr>
-                            <tr>
-                                <td>JAI Image I/O Tools</td>
-                                <td><a href="https://downloads.openmicroscopy.org/bio-formats/5.5.0/artifacts/jai_imageio.jar">jai_imageio.jar</a></td>
-                            </tr>
-                            <tr>
-                                <td>TurboJPEG</td>
-                                <td><a href="https://downloads.openmicroscopy.org/bio-formats/5.5.0/artifacts/turbojpeg.jar">turbojpeg.jar</a></td>
-                            </tr>
-                        </tbody>
-                    </table>
+        
+        {% include back-to-top.html %}
+        
+        <!-- begin CTA -->
+        <hr class="whitespace">
+        <header class="marketing-hero" style="background-image: url('{{ site.baseurl }}/img/bg/bio-formats.jpg');">
+            <div class="row">
+                <div class="small-12 columns bg-image-text-container-cta text-center">
+                    <p class="hero-subheader">All components available for this version of Bio-Formats are available here:</p>
+                    <a href="http://downloads.openmicroscopy.org/bio-formats/?C=M;O=D" class="large btn-red button sites-button hide-for-small-only" style="text-shadow: none;">Download Components</a>
```

Comment:
I'm fine with deleting the artifact name for the OMERO ones; just say the official word :)